test/ext/ws.js

```diff
@@ -88,11 +88,36 @@ describe("web-sockets extension", function () {
         this.messages.length.should.equal(1);
     })
 
-    it('handles message from the server', function () {
+    it('handles message from the server (blob, default)', function () {
+        htmx.config.wsBinaryType = "blob";
         var div = make('<div hx-ext="ws" ws-connect="ws://localhost:8080"><div id="d1">div1</div><div id="d2">div2</div></div>');
         this.tickMock();
 
-        this.socketServer.emit('message', "<div id=\"d1\">replaced</div>");
+        this.socketServer.emit('message', new Blob(["<div id=\"d1\">replaced</div>"]));
+
+        setTimeout(function() {
+            byId("d1").innerHTML.should.equal("replaced");
+            byId("d2").innerHTML.should.equal("div2");
+        }, 10);
+    })
+
+    // pulled from: https://developer.chrome.com/blog/how-to-convert-arraybuffer-to-and-from-string/
+    var str2ab = function(str) {
+        var buf = new ArrayBuffer(str.length*2); // 2 bytes for each char
+        var bufView = new Uint16Array(buf);
+        for (var i=0, strLen=str.length; i < strLen; i++) {
+            bufView[i] = str.charCodeAt(i);
+        }
+        return buf;
+    }
+
+    it('handles message from the server (arraybuffer)', function () {
+        htmx.config.wsBinaryType = "arraybuffer";
```

Comment:
I think this value should be reset to default between tests. Best way to do that is setting it to default at the top of the file in `afterEach` call